### ISSUE TITLE
Application timeout should be made accessible

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2252]
**Describe the solution you'd like**
Session timeout modal on the public application should be announced to screen readers

**What is the acceptance criteria/definition of done?**
- Add role "alertdialog" and aria-labelledby=title as well as an ID on the modal title set to "title" 

**QA Review Instructions**
- Enable VoiceOver
- Navigate to the short form application
- Allow the app to timeout
- Modal should be announced as alertdialog

https://cccaccessibility.org/web-1/web-developer-tutorials/accessible-timeouts
